testing/trino-test-jdbc-compatibility-old-driver/bin/run_tests.sh

```diff
@@ -10,11 +10,14 @@ maven_run_tests="${maven} clean test -Dair.check.skip-all=true -Dmaven.javadoc.s
 
 current_version=$(${maven} help:evaluate -Dexpression=project.version -q -DforceStdout)
 previous_released_version=$((${current_version%-SNAPSHOT}-1))
-first_tested_version=312
+first_tested_version=351
 # test n-th version only
 version_step=3
 
 echo "Current version: ${current_version}"
+
+(( previous_released_version >= first_tested_version )) || exit 0
```

Comment:
This should use explicit `if`. `||` for control flow does not add to readability